pcsx2/IopCounters.cpp

```diff
@@ -55,15 +55,20 @@ u8 psxvblankgate = 0;
 
 // used to disable targets until after an overflow
 #define IOPCNT_FUTURE_TARGET (0x1000000000ULL)
+#define IOPCNT_MODE_WRITE_MSK 0x63FF
+#define IOPCNT_MODE_FLAG_MSK 0x1800
 
 #define IOPCNT_ENABLE_GATE (1 << 0)  // enables gate-based counters
 #define IOPCNT_MODE_GATE (3 << 1)    // 0x6  Gate mode (dependant on counter)
-#define IOPCNT_MODE_RESET (1 << 3)   // 0x8  resets the counter on target (if interrupt only?)
+#define IOPCNT_MODE_RESET_CNT (1 << 3)   // 0x8  resets the counter on target (if interrupt only?)
```

Comment:
Would be nice to align the comments again.

Reply:
Id did, then I undid it cos I thought you'd get pissy about the random tab/space combinations xD